apps/surfer/src/components/SurferContainer.tsx

```diff
@@ -10,12 +13,24 @@ export const SurferContainer = forwardRef<
   HTMLDivElement,
   {
     isLoading: boolean;
+    isExpanded?: boolean;
+    toggleExpanded?: (nextState: boolean) => void;
     flex?: string;
     height?: string;
   }
->(({ isLoading, flex, height }, ref) => (
-  <>
+>(({ isLoading, flex, height, isExpanded, toggleExpanded }, ref) => (
+  <Flex flexDirection="column" justifyContent="space-between" fullHeight gap={tokens.spacingS} style={{ overflow: 'hidden' }}>
```

Comment:
Without `overflow: hidden` scrollbars are out of control. This way it looks way better without impairing usability.

Reply:
Style fixes in other integrations (e.g. gdocs or Wordpress) could result in Contentful integration being partially unusable due to `overflow: hidden`. If it's very challenging to align everything properly, we can leave it as is, but `overflow: auto` is a proper thing to have here

Reply:
Yes, however given our current setup and time constraints, I'd suggest we leave it like that. Here's the issue with `overflow: auto`, we get those double scrollbars (because the content of the iframe already has them):

![image](https://github.com/surferseo/contentful/assets/28512248/312115a2-498e-4459-b8c3-3411256337ea)

It's probably a byproduct of embedding this pre-existing extension inside another app, what we need to do it better is more control over the width of the content inside the iframe, which we don't have right now.